.env.example

```
@@ -1,3 +1,4 @@
-VITE_SUPABASE_URL=
-VITE_SUPABASE_ANON_KEY=
-VITE_API_URL=
\ No newline at end of file
+# Copy this file to .env and populate it with your actual project values from the Supabase dashboard
+SUPABASE_URL=https://your-project.supabase.co
+SUPABASE_ANON_KEY=your-anon-key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SUPABASE_ANON_KEY key should go before the SUPABASE_URL key

(UnorderedKey)